easybuild/framework/easyblock.py

```diff
@@ -1301,6 +1301,12 @@ def make_module_req(self):
                     # only use glob if the string is non-empty
                     if path and not self.dry_run:
                         paths = sorted(glob.glob(path))
+                        if key in ('PATH', 'CPATH', 'LIBRARY_PATH', 'LD_LIBRARY_PATH') and len(paths) == 1:
```

Comment:
@bartoldeman I don't think there's a need to only do this when there's a single directory being considered?

see https://github.com/ComputeCanada/easybuild-framework/pull/18